vignettes/eatPrep.Rmd

```diff
@@ -57,38 +85,60 @@ Information/metadata about items is stored in many different ways, e.g., databas
 
-Example: inputList, inputMinimal
-
 `inputMinimal` contains the bare minimum for all functions in `eatPrep` to work, `inputList` contains additional metadata about the items (such as information about the task's content).
 
+<!--Example: inputList ? -->
+
+Example: inputMinimal
+
+```{r tempsetup}
+inputMinimal <- list(units = inputList$units[ -nrow(inputList$units), c("unit", "unitAggregateRule")],
+                     subunits = inputList$subunits[, c("unit", "subunit","subunitRecoded")],
+                     values = inputList$values[ , c("subunit", "value", "valueRecode", "valueType")],
+                     unitRecodings = inputList$unitRecodings[ , c("unit", "value", "valueRecode", "valueType")],
+                     blocks = inputList$blocks,
+                     booklets = inputList$booklets,
+                     rotation = inputList$rotation)
+```
+
 ### Input tables
 Let's look at the tables from bottom to top, starting with items and their values. In the values sheet, we define values and recode values for each subitem.
 
-
 Here we see 3 items (called units) from `inputList` bzw. `inputMinimal`.
-One mc (item 1), one short response (item 2), one item with 3 subitems short response (item 12)
+One mc (item 1), one short response (item 5), one item with 3 subitems short response (item 12)
 
 ```{r inputList}
 items <- c("I01", "I05", "I12")
 subitems <- c("I01", "I05", "I12a", "I12b", "I12c")
 
 kable(inputMinimal$units[which(inputMinimal$units$unit %in% items), ])
-kable(inputMinimal$subunits[which(inputMinimal$subunits$subunit %in% subitems), ])
-kable(inputMinimal$values[which(inputMinimal$values$subunit %in% subitems), ])
-kable(inputMinimal$unitRecodings)
 
 ```
 
-The values sheet contains all possible values for each subitem and scoring information for each value (e.g., whether it is true, false, or missing). There are several types of missing values.
 
---> talk more about values in the data for different item types (mc, cr) and about missing values in the data. missing values do not need to be the same for each variable in the data.
 
+The **values sheet** contains all possible values for each subitem and scoring information for each value (e.g., whether it is true, false, or missing). There are several types of missing values.
 
-The subunits sheet contains information about the relationship between subitems and items. This information is used to aggregate subitems to items. If recoded subitems should be named differently than the unrecoded variables in the original data, this can be specified via `subunitRecoded`. The only type of aggregation currently supported by `eatPrep` is to count the number of correct subitems per item.
+```{r values sheet}
+kable(inputMinimal$values[which(inputMinimal$values$subunit %in% subitems), ])
+```
+
+<!-- talk more about values in the data for different item types (mc, cr) and about missing values in the data. missing values do not need to be the same for each variable in the data.-->
```

Comment:
do you mean similar to the table for missing types? or more detailed on when to give what kind of value?